main.go

```diff
@@ -475,8 +475,14 @@ func TextDocumentDidOpen(ctx context.Context, vs lsp.DidOpenTextDocumentParams)
 	return nil
 }
 
+func Shutdown(ctx context.Context, vs lsp.None) error {
+	os.Remove(tempFile.Name())
+	return nil
+}
+
 func Exit(ctx context.Context, vs lsp.None) error {
 	os.Remove(tempFile.Name())
+	os.Exit(0)
```

Comment:
Since we specify an `error` to return, how are we going to return it if we do `os.Exit(0)`? We may want to remove the error return value...